hi-ml/src/health_ml/utils/regression_test_utils.py

```diff
@@ -25,6 +26,45 @@
 TEXT_FILE_SUFFIXES = [".txt", ".json", ".html", ".md"]
 
 
+def compare_dictionaries(expected: Dict[str, Any], actual: Dict[str, Any], tolerance: Optional[float] = 1e-5) -> None:
+    """
+    Function to give more clarity on the difference between two dictionaries.
+
+    :param expected: The first dictionary to compare
+    :param actual: The second dictionary to compare
+    :param tolerance: The tolerance to allow when comparing numeric values, defaults to 1e-5
+    """
+    def _check_values_match(expected_value: Any, actual_value: Any, tolerance: Optional[float] = 1e-5) -> None:
+        if type(actual_value) in [float, int] and type(expected_value) in [float, int]:
+            if abs(actual_value - expected_value) > tolerance:
```

Comment:
the argument that you pass in is a relative tolerance, you are using it as an absolute here